lib/storefront/award-biginnovation.svg

```diff
@@ -1,12 +1,101 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Generator: Adobe Illustrator 19.2.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<!-- Generator: Adobe Illustrator 19.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
Since so many other icons keep this stuff and it gets stripped out when minified, maybe it's not worth it?